addons/website/tests/test_performance.py

```diff
@@ -269,9 +275,11 @@ def test_40_perf_sql_queries_page_multi_level_menu(self):
             'website': 1,
             'ir_ui_view': 1,
             # Check if `view.track` to track visitor or not
+            'res_currency': 1,
```

Comment:
This need to be changed, we are not going to add a sql query to the currency table for every single page load in the website :grimacing: 
We spent years trying to reduce this number, even for very low level and important table, adding a query to this not so important table (it's useless for website) is surely a no go.

Reply:
It's fine where it is :+1: 
Ty for the perf change